lab-lee/README.md

```diff
@@ -0,0 +1,15 @@
+# Purpose of this Project
+
+The purpose of this project is to allow a user to make either a HTTP GET or HTTP POST request and have the response displayed using the cowsay API.
+
+GET Requests:
+        http localhost:3000/ //will respond with "hello world" with no cowsay
+        http localhost:3000/cowsay text=='[insert text]' : will respond with cowsay //will
```

Comment:
You should either specify that this is a request formatted using HTTPie (as other clients might not have queries formatted in this way) or just tell the user that they need to send a GET request to localhost:3000/cowsay with a query string containing a property of text with a value of whatever they want the cow to say... or something to that effect.